### PR TITLE
feat(blog): add post for Seth Goings

### PR DESCRIPTION
**Pull request by carmstrong**

### CODE REVIEW

File: _posts/2015-07-15-welcome-seth-goings.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  - name: description
+    content: Please join us in welcoming Seth Goings to the Deis project!
+  - name: keywords
+    content: deis, docker, paas, open source, seth goings
+---
+
+Seth is a software engineer who has always despised how much time, effort, sweat,
+and tears are involved in producing high quality software projects. He'd rather
+spend time running away from snarling Chihuahuas, riding his bicycle off into
+the sunset, cooking up an all kitchen-appliances-necessary dinner, or remodeling
```
Comment (sgoings) [inline _posts/2015-07-15-welcome-seth-goings.md]: all-kitchen-appliances-necessary (see dash between all and kitchen)

File: _posts/2015-07-15-welcome-seth-goings.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+his house at 3 in the morning than stepping through a testing spreadsheet
```
Comment (sgoings) [inline _posts/2015-07-15-welcome-seth-goings.md]: "3 in the morning" -> "3AM" seems to read faster

**Comment by bacongobbler** [comment]
LGTM!

**Comment by sgoings** [comment]
:+1:

**Comment by mboersma** [comment]
LGTM.

**Pull request merged**